ansible/roles/ocp4-workload-rhte-analytics_data_ocp_workshop/tasks/workload.yml

```diff
@@ -2,189 +2,184 @@
+#
+#- include_tasks: per_user_post_operator_workload.yml
+#  loop: "{{ range(1, num_users | int + 1, 1) | list }}"
+#  loop_control:
+#    loop_var: user_num
+#
+## parallel-ish validation of deployments
+#- name: Wait for various deploymentconfigs to deploy
+#  k8s_facts:
+#    api_version: apps.openshift.io/v1
+#    kind: DeploymentConfig
+#    name: "{{ item.1 }}"
+#    namespace: "opendatahub-user{{ item.0 }}"
+#  register: deployment_out
+#  until:
+#    - deployment_out is defined
+#    - deployment_out.resources is defined
+#    - deployment_out.resources[0] is defined
+#    - deployment_out.resources[0].status is defined
+#    - deployment_out.resources[0].status.readyReplicas is defined
+#    - deployment_out.resources[0].status.readyReplicas | int >= 1
+#  loop: "{{ (range(1, num_users | int + 1, 1) | list) | product(validation_deploymentconfigs) | list }}"
+#  retries: 18
+#  delay: 10
+#
+#- name: Wait for various deployments to deploy
+#  k8s_facts:
+#    api_version: extensions/v1beta1
+#    kind: Deployment
+#    name: "{{ item.1 }}"
+#    namespace: "opendatahub-user{{ item.0 }}"
+#  register: deployment_out
+#  until:
+#    - deployment_out is defined
+#    - deployment_out.resources is defined
+#    - deployment_out.resources[0] is defined
+#    - deployment_out.resources[0].status is defined
+#    - deployment_out.resources[0].status.readyReplicas is defined
+#    - deployment_out.resources[0].status.readyReplicas | int >= 1
+#  loop: "{{ (range(1, num_users | int + 1, 1) | list) | product(validation_deployments) | list }}"
+#  retries: 18
+#  delay: 10
+#
+#- name: Wait for various statefulsets to deploy
+#  k8s_facts:
+#    api_version: apps/v1
+#    kind: StatefulSet
+#    name: "{{ item.1 }}"
+#    namespace: "opendatahub-user{{ item.0 }}"
+#  register: deployment_out
+#  until:
+#    - deployment_out is defined
+#    - deployment_out.resources is defined
+#    - deployment_out.resources[0] is defined
+#    - deployment_out.resources[0].status is defined
+#    - deployment_out.resources[0].status.readyReplicas is defined
+#    - deployment_out.resources[0].status.readyReplicas | int >= 1
+#  loop: "{{ (range(1, num_users | int + 1, 1) | list) | product(validation_statefulsets) | list }}"
+#  retries: 18
+#  delay: 10
+#
+#- name: Wait for various 3-member statefulsets to deploy
+#  k8s_facts:
+#    api_version: apps/v1
+#    kind: StatefulSet
+#    name: "{{ item.1 }}"
+#    namespace: "opendatahub-user{{ item.0 }}"
+#  register: deployment_out
+#  until:
+#    - deployment_out is defined
+#    - deployment_out.resources is defined
+#    - deployment_out.resources[0] is defined
+#    - deployment_out.resources[0].status is defined
+#    - deployment_out.resources[0].status.readyReplicas is defined
+#    - deployment_out.resources[0].status.readyReplicas | int >= 3
+#  loop: "{{ (range(1, num_users | int + 1, 1) | list) | product(validation_3statefulsets) | list }}"
+#  retries: 18
+#  delay: 10
```

Comment:
If this code is not needed, please delete it from the role. No need to keep history with comments in version control.